include/aws/common/hash_table.h

```diff
@@ -270,7 +270,7 @@ int aws_hash_table_create(
  * element was found, or 1 is a new element was created.
  *
  * Returns AWS_OP_SUCCESS if an item was found or created.
- * Raises AWS_ERROR_OOM if hash table expansion was required and memory
+ * Raises AWS_ERROR_OOM if hash table expansion was required and ran out of memory
```

Comment:
Probably out of scope for this pr, but do we even have to worry about this anymore? iirc mike said something about just crashing on OOM now right?

Reply:
I don't want to change the behavior of others from this pr. But, just to update the doc, which was inaccurate.